tests/test_utils.cpp

```diff
@@ -60,44 +60,27 @@ void printIssues(const libcellml::LoggerPtr &l, bool headings, bool causes, bool
     }
 }
 
-void prettyPrintModelToStdout(libcellml::ModelPtr &model, bool includeMaths)
-{
-    std::string spacer = "    ";
-
-    std::cout << " MODEL: '" << model->name() << "'";
-    if (model->id() != "") {
-        std::cout << ", id: '" << model->id() << "'";
-    }
-    std::cout << std::endl;
-
-    std::cout << spacer << "UNITS: " << model->unitsCount() << " custom units" << std::endl;
-    for (size_t u = 0; u < model->unitsCount(); ++u) {
-        std::cout << spacer << spacer << "[" << u << "]: " << model->units(u)->name() << std::endl;
-    }
+static const std::string INDENT = "    ";
 
-    std::cout << spacer << "COMPONENTS: " << model->componentCount() << " components" << std::endl;
-    for (size_t c = 0; c < model->componentCount(); ++c) {
-        auto component = model->component(c);
-        prettyPrintComponentToStdout(component, c, spacer + spacer, includeMaths);
-    }
-}
-
-void prettyPrintComponentToStdout(const libcellml::ComponentPtr &component, size_t const c, std::string const spacer, bool includeMaths)
+void printComponent(const libcellml::ComponentPtr &component, size_t c, const std::string &indent, bool includeMaths)
```

Comment:
This was recently changed to 'prettyPrint*' are we done with that now?

Reply:
The utility was originally named `printComponent()` and I don't see why it was renamed to something unnecessarily long and that I would never remember, hence I reverted the name of the method to what it is in `develop`.